### ISSUE TITLE
Some images are all black, is this normal?

### ISSUE THREAD

**Issue by zjh908491372** [facebookresearch/InterHand2.6M#134]
Some images are all black, is this normal?
Is this the problem of the data set itself or the wrong way of decompression？
# =================================================================#
InterHand2.6M_5fps_batch1/images/train/Capture0/0135_palmdown/cam410067/image37273.jpg
InterHand2.6M_5fps_batch1/images/train/Capture0/0135_palmdown/cam410069/image37273.jpg
InterHand2.6M_5fps_batch1/images/train/Capture0/0135_palmdown/cam410208/image37273.jpg
InterHand2.6M_5fps_batch1/images/train/Capture0/0135_palmdown/cam410219/image37273.jpg
InterHand2.6M_5fps_batch1/images/train/Capture0/0135_palmdown/cam410226/image37273.jpg
InterHand2.6M_5fps_batch1/images/train/Capture0/0130_middletip/cam400008/image35646.jpg
InterHand2.6M_5fps_batch1/images/train/Capture0/0130_middletip/cam400010/image35646.jpg
InterHand2.6M_5fps_batch1/images/train/Capture0/0130_middletip/cam400059/image35646.jpg
InterHand2.6M_5fps_batch1/images/train/Capture0/0148_index_point/cam410050/image42342.jpg
InterHand2.6M_5fps_batch1/images/train/Capture0/0013_surfer/cam410049/image4973.jpg
InterHand2.6M_5fps_batch1/images/train/Capture0/0013_surfer/cam410049/image4991.jpg
InterHand2.6M_5fps_batch1/images/train/Capture0/0013_surfer/cam410049/image5009.jpg
InterHand2.6M_5fps_batch1/images/train/Capture0/0013_surfer/cam410049/image5027.jpg
InterHand2.6M_5fps_batch1/images/train/Capture0/0013_surfer/cam400059/image5045.jpg
InterHand2.6M_5fps_batch1/images/train/Capture0/0013_surfer/cam410049/image5045.jpg
InterHand2.6M_5fps_batch1/images/train/Capture0/0013_surfer/cam410220/image5045.jpg
InterHand2.6M_5fps_batch1/images/train/Capture0/0013_surfer/cam400008/image5063.jpg
InterHand2.6M_5fps_batch1/images/train/Capture0/0013_surfer/cam400042/image5063.jpg
InterHand2.6M_5fps_batch1/images/train/Capture0/0013_surfer/cam400059/image5063.jpg
InterHand2.6M_5fps_batch1/images/train/Capture0/0013_surfer/cam410049/image5063.jpg
InterHand2.6M_5fps_batch1/images/train/Capture0/0013_surfer/cam410050/image5063.jpg
InterHand2.6M_5fps_batch1/images/train/Capture0/0013_surfer/cam410053/image5063.jpg
InterHand2.6M_5fps_batch1/images/train/Capture0/0013_surfer/cam410220/image5063.jpg
InterHand2.6M_5fps_batch1/images/train/Capture0/0013_surfer/cam410238/image5063.jpg
# =================================================================#

**Comment by mks0601** [comment]
Yes it is normal, and that is due to the camera failure. We filtered such black images, but could not filter all of them as there are too many images (2.6M images). As the number of all images is very large (2.6M), we observed that such black images affect the final error marginally

**Comment by zjh908491372** [comment]
Got it.
